packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -46,7 +46,7 @@ export const PaymentErrorView = ({
             <button
               data-testid="retry-link"
               className="button retry-link"
-              onClick={() => onRetry()}
+              onClick={onRetry}
```

Comment:
it's safer to leave this wrapped otherwise the event argument from onClick will get passed to onRetry. it _usually_ won't matter but can lead to subtle bugs like it did here https://github.com/mozilla/fxa/pull/7514

Reply:
That's an interesting tip.  Thanks!

Reply:
@chenba Should I hold off on merging then?

Reply:
Either way is fine with me.  Easier to just not merge and close?